### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Guillaume Trueb

### DISCUSSION THREAD

**Discussion by grinlemon** [heig-vd-dai-course/heig-vd-dai-course#123] [Show and tell]
https://github.com/truebguillaume/DAI-Docker-Compose-App
<img width="904" alt="image" src="https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/73121793/8e046253-96ba-43b8-b90b-91f318dbfe79">

**Comment by ludelafo** [comment]
The Docker Compose file seems good to me!

However, I would add a gitignore file, as mentioned in the practice content: <https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/10-docker-and-docker-compose/COURSE_MATERIAL.md#share-your-docker-compose-application>.

The config and the data of the File Browser application is not meant to be shared with the rest of the world. The database contains the password used to access the administration panel and the data should stay private as well. As a bonus, I would add the `DS_Store` file to the gitignore as well to avoid noise in your repository.
